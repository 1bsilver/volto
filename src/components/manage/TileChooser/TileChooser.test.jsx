import React from 'react';
import renderer from 'react-test-renderer';
import TileChooser from './TileChooser';

test('renders a TileChooser component', () => {
  const component = renderer.create(
    <TileChooser onMutateTile={() => {}} currentTile="thetileid" />,
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
