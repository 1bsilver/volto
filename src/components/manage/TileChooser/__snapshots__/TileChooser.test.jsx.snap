// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a TileChooser component 1`] = `
<div
  className="tiles-chooser"
>
  <div
    className="accordion ui fluid styled form"
  >
    <div
      className="active title"
      onClick={[Function]}
    >
      mostUsed
      <div
        className="accordion-tools"
      >
        <svg
          className="icon"
          dangerouslySetInnerHTML={
            Object {
              "__html": undefined,
            }
          }
          onClick={null}
          style={
            Object {
              "fill": "currentColor",
              "height": "20px",
              "width": "auto",
            }
          }
          viewBox=""
          xmlns=""
        />
      </div>
    </div>
    <div
      className="content active tiles-list"
    >
      <div
        aria-hidden={false}
        className="rah-static rah-static--height-auto"
        style={
          Object {
            "height": "auto",
            "overflow": "visible",
          }
        }
      >
        <div
          style={
            Object {
              "WebkitTransition": "opacity 500ms ease 0ms",
              "transition": "opacity 500ms ease 0ms",
            }
          }
        >
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              image
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              video
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="title"
      onClick={[Function]}
    >
      text
      <div
        className="accordion-tools"
      >
        <svg
          className="icon"
          dangerouslySetInnerHTML={
            Object {
              "__html": undefined,
            }
          }
          onClick={null}
          style={
            Object {
              "fill": "currentColor",
              "height": "20px",
              "width": "auto",
            }
          }
          viewBox=""
          xmlns=""
        />
      </div>
    </div>
    <div
      className="content tiles-list"
    >
      <div
        aria-hidden={true}
        className="rah-static rah-static--height-zero"
        style={
          Object {
            "height": 0,
            "overflow": "hidden",
          }
        }
      >
        <div
          style={
            Object {
              "WebkitTransition": "opacity 500ms ease 0ms",
              "opacity": 0,
              "transition": "opacity 500ms ease 0ms",
            }
          }
        >
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              title
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              description
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              text
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="title"
      onClick={[Function]}
    >
      media
      <div
        className="accordion-tools"
      >
        <svg
          className="icon"
          dangerouslySetInnerHTML={
            Object {
              "__html": undefined,
            }
          }
          onClick={null}
          style={
            Object {
              "fill": "currentColor",
              "height": "20px",
              "width": "auto",
            }
          }
          viewBox=""
          xmlns=""
        />
      </div>
    </div>
    <div
      className="content tiles-list"
    >
      <div
        aria-hidden={true}
        className="rah-static rah-static--height-zero"
        style={
          Object {
            "height": 0,
            "overflow": "hidden",
          }
        }
      >
        <div
          style={
            Object {
              "WebkitTransition": "opacity 500ms ease 0ms",
              "opacity": 0,
              "transition": "opacity 500ms ease 0ms",
            }
          }
        >
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              image
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              video
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="title"
      onClick={[Function]}
    >
      common
      <div
        className="accordion-tools"
      >
        <svg
          className="icon"
          dangerouslySetInnerHTML={
            Object {
              "__html": undefined,
            }
          }
          onClick={null}
          style={
            Object {
              "fill": "currentColor",
              "height": "20px",
              "width": "auto",
            }
          }
          viewBox=""
          xmlns=""
        />
      </div>
    </div>
    <div
      className="content tiles-list"
    >
      <div
        aria-hidden={true}
        className="rah-static rah-static--height-zero"
        style={
          Object {
            "height": 0,
            "overflow": "hidden",
          }
        }
      >
        <div
          style={
            Object {
              "WebkitTransition": "opacity 500ms ease 0ms",
              "opacity": 0,
              "transition": "opacity 500ms ease 0ms",
            }
          }
        >
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              hero
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              maps
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              html
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              className="ui basic icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "36px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
              table
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
